src/pylife/materiallaws/__init__.py

```diff
@@ -27,5 +28,9 @@
     'true_strain',
     'true_fracture_stress',
     'true_fracture_strain',
-    'WoehlerCurve'
+    'WoehlerCurve',
+    'Hookeslaw1D',
+    'Hookeslaw2Dstrain',
+    'Hookeslaw2Dstress',
+    'Hookeslaw3D'
```

Comment:
Generally I would suggest to rename the classes to `HookesLaw*` with capital `L`. Imagine we'd implement some law invented by a guy named "Cole" :smile: